From 35011a5da841196ada6e9c73a251a09f8df5c1c3 Mon Sep 17 00:00:00 2001
From: Brian Silverman <bsilverman@iveia.com>
Date: Mon, 1 Feb 2021 15:33:42 -0500
Subject: [PATCH] banner: added additional CRLF

Xilinx banner includes a CRLF at the front, that often gets lost on the
console.  Add more to make start of boot more visible.

---
 lib/sw_apps/zynqmp_fsbl/src/xfsbl_main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/sw_apps/zynqmp_fsbl/src/xfsbl_main.c b/lib/sw_apps/zynqmp_fsbl/src/xfsbl_main.c
index d38f799e91..9f496dec65 100644
--- a/lib/sw_apps/zynqmp_fsbl/src/xfsbl_main.c
+++ b/lib/sw_apps/zynqmp_fsbl/src/xfsbl_main.c
@@ -342,6 +342,7 @@ void XFsbl_PrintFsblBanner(void )
 	 */
 #if !defined(XFSBL_PERF) || defined(FSBL_DEBUG) || defined(FSBL_DEBUG_INFO) \
 			|| defined(FSBL_DEBUG_DETAILED)
+	XFsbl_Printf(DEBUG_PRINT_ALWAYS, "\r\n\r\n");
 	XFsbl_Printf(DEBUG_PRINT_ALWAYS,
                  "Xilinx Zynq MP First Stage Boot Loader \n\r");
 	XFsbl_Printf(DEBUG_PRINT_ALWAYS,
